### PR TITLE
RBAC for CIS v2

### PR DESCRIPTION
**Pull request by prachidamle**

### CODE REVIEW

**Comment by catherineluse** [comment]
Lgtm

File: content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+| ClusterRole created by chart | Default K8s ClusterRole  | Permissions given with Role
+| ------------------------------| ---------------------------| ---------------------------|
+| `cis-admin` | `admin`| Ability to CRUD clusterscanbenchmarks, clusterscanprofiles, clusterscans, clusterscanreports CR
+| `cis-edit`| `edit` | Ability to CRUD clusterscanbenchmarks, clusterscanprofiles, clusterscans, clusterscanreports CR
+| `cis-view` | `view `| Ability to List(R) clusterscanbenchmarks, clusterscanprofiles, clusterscans, clusterscanreports CR
+
+Rancher will continue to use cluster-owner, cluster-member, project-owner, project-member, etc as role names, but these default k8s roles will determine access to CIS feature.
+By default only cluster-owner role will have ability to use `rancher-cis-benchmark` feature.
+But the above ClusterRoles can be granted to cluster-member, project-owner, project-member users if a cluster-owner wants to share access.
```
Comment (cbron) [inline content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: We need to explain to customers what rancher roles get what permissions. What CIS permissions does a project-member of a random NS get ? It's not obvious to users what rancher roles "cis-edit" translates too.

Comment (cbron) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: Also this is incorrect: 
> But the above ClusterRoles can be granted to cluster-member, project-owner, project-member users if a cluster-owner wants to share access.

These roles are automatically granted via aggregation. For example cluster-owner gets admin privileges I believe in all the NS's he owns, so he automatically inherits the cis-admin privileges in all those Namespaces. We aren't expecting cluster owners to hand these out manually.

Comment (prachidamle) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: @cbron I dont think any of the other rancher roles cluster-member, project-owner, project-member gets anything by default - Unless we automatically assign default k8s permissions to these - atleast I dont know that. 
So if they are not auto-assigned the k8s defaults, they will have to be manually handed over by the admin.

Comment (prachidamle) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: @cbron will be good to know this - what k8s defaults are assigned to cluster-member, project-owner, project-member by rancher?

Comment (cbron) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: It happens automatically via: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
I believe our project-owner gets admin in their NS's, project-member gets edit

File: content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: RBAC
+weight: 3
+---
+
+This section describes the permissions required to use the rancher-cis-benchmark App.
+
+The rancher-cis-benchmark is a cluster-admin only feature by default.
+
+However, the `rancher-cis-benchmark` chart installs three default `ClusterRoles`:
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:10]: This list does not appear as a list in the UI actual docs

File: content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: I don't understand. What change needs to be made to the docs? are we just saying that only cluster owners can run scans and see the reports?

Comment (prachidamle) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: @cbron I don't see that a project-owner or project-member get access to CIS resources even if the cis-operator-system NS is moved to that project. Is there a specific step to test this? 
Only cluster-owner and global-admin have admin access by default. All other rancher roles do not  see CIS resources
by default unless they are granted the cis-admin/cis-edit/cis-view roles. 
So is the current documented rbac info correct?

**Comment by prachidamle** [comment]
@catherineluse we should document the following:

```
By default few ClusterScanBenchmarks and ClusterScanProfiles are installed as part of the rancher-cis-benchmark chart. If a user edits these default benchmarks or profiles, the next chart update will reset them back. So it is advisable for users to not edit the default ClusterScanBenchmarks or ClusterScanProfiles. 
Users can clone the default ClusterScanProfiles to create new custom profiles.
```

File: content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md (hunk 1 above)
Comment (prachidamle) [reply content/rancher/v2.x/en/cis-scans/2.5.x/rbac/_index.md:40]: The reason why the role aggregation does not work for CIS Custom Resources is that all CIS CRs are cluster-scoped. 
So only the cluster-owner Rancher role gets the cis-admin role via aggregation.
For rest of the Rancher roles (cluster-member, project-owner or project-member), the admin will have to manually create CRTBs to grant the CIS roles to these users.

**Pull request merged**